test/fish_prompt_test.fish

```diff
@@ -23,44 +23,57 @@ function assert \
   end
 end
 
+function os_prompt_path_prefix
+  set os_name (uname -s)
+
+  if [ $os_name = 'Darwin' ]
+    echo '/p/t'
+  else
+    echo '/t'
+  end
+end
+
 set path '/tmp/fish-theme-mean-test'
+set prompt_path_prefix (os_prompt_path_prefix)
```

Comment:
Does this leaks to our environment as well?
Thinking out loud.

Reply:
@lsunsi,  for some reason, it doesn't. Maybe fisherman (or fish) ignores test files??

Reply:
Yeah, just tried it. Got curious on why it doesn't set it.
Moving on